jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ChipsTests.java

```diff
@@ -32,67 +39,126 @@ public class ChipsTests extends TestsInit {
     private static final String LEMON = "Lemon";
     private static final String LIME = "Lime";
 
-
-    @BeforeMethod(alwaysRun = true)
+    @BeforeMethod
     public void before() {
-        shouldBeLoggedIn();
-        angularPage.shouldBeOpened();
+        chipsPage.open();
+        waitCondition(() -> chipsPage.isOpened());
+        chipsPage.checkOpened();
     }
 
-    @Test
+    @Test(description = "Test verifies that chip group is selectable")
     public void basicChipsTest() {
-        basicChipsRow.show();
-        basicChipsRow.is().displayed();
-        basicChipsRow.is().assertChipsIsEnabled();
-        basicChipsRow.clickChipsByTextValue(ONEFISH);
-        basicChipsRow.clickChipsByTextValue(TWOFISH);
-        basicChipsRow.clickChipsByTextValue(PRIMARYFISH);
-        basicChipsRow.clickChipsByTextValue(ACCENTFISH);
-
-        basicSelectedValue.has().text(format("You clicked: %s", ACCENTFISH));
+        basicChips.show();
+        basicChips.is().displayed();
+        basicChips.selectByText(TWOFISH);
+        basicChips.selectByText(PRIMARYFISH);
+        basicChips.selectByText(WARNFISH);
+
+        ChipsPage.basicSelectedValue.has().text(format("You clicked: %s", WARNFISH));
     }
 
-    @Test
+    @Test(description = "Test verifies that chip group is vertical")
     public void stackedChipsTest() {
-        stackedChipsList.show();
-        stackedChipsList.is().displayed();
-        stackedChipsList.is().assertChipsIsEnabled();
-        stackedChipsList.clickChipsByTextValue(NONE);
-        stackedChipsList.clickChipsByTextValue(PRIMARY);
-        stackedChipsList.clickChipsByTextValue(ACCENT);
-        stackedChipsList.clickChipsByTextValue(WARN);
+        stackedChips.show();
+        stackedChips.is().displayed();
+        stackedChips.is().vertical();
+        stackedChips.selectByText(NONE);
+        stackedChips.selectByText(PRIMARY);
```

Comment:
You select something and do not check that somethings is changed after, it is not a good test